documents/description.md

```diff
@@ -1,9 +1,42 @@
 # Project description
 
 ## Motivation
+I am not really trying to address and existing problem with this project as much as I am trying to create a neat tool to help (approximately) middle-school aged students to write and check their poetry. I want the project to help students write poetry which has known meter and rhyme scheme, which would hopefully encourage them to be creative and write more. I think this is interesting because nothing like it really exists, and it seems to be something with a fairly simple MVP with lots of extensibility, which makes it (in my opinion) a really cool project.
+
+I think a DSL is a good idea because it will help with (minor) variability in structured poetry. For example, you would be able to specify a "AABB" rhyme scheme, or a certain number of syllables, and in that way create a new form of structured poem; alternatively, you could use a known form like a sonnet or limerick, and check if you are appropriately following the known form.
 
 ## Language domain
+The domain for this language is in poetry-writing, specifically in metered poetry. What I think makes this project cool is that nothing (that I've found) does anything remotely like this. Of course, there are rhyming and syllable suggestion/checking sites aplenty online, and lots of APIs that help with various aspects of language from synonyms to spellchecking, there isn't really anything out there quite like what I want to do. 
```

Comment:
Is this really the domain? You talk about this program helping children learn how to write poetry, so it seems like the domain might fit more on the teaching side. Think about programs for image manipulation. Photoshop is in that domain but is not at all beginner friendly. Paint on the other hand (or a teaching program, don't know the domain that much), is much more beginner friendly and allows those users to iteratively deepen their understanding of the topic. So while they accomplish similar tasks, their domain is different since they know their target users are different. So would an expert poetry writer use your program? Probably not, so its important to specify the exact domain so that you are sure you are targeting the specifics of the DSL to the right end user.